cmd/common_konnect.go

```diff
@@ -52,7 +52,7 @@ func syncKonnect(ctx context.Context,
 
 	// initialize kong client
 	kongClient, err := utils.GetKongClient(utils.KongClientConfig{
-		Address:    konnect.BaseURL() + "/api/control_planes/" + kongCPID,
+		Address:    konnectConfig.Address + "/api/control_planes/" + kongCPID,
```

Comment:
In the past I got bitten by kong.Client exploding when the kong URL provided by the user ends with a `/`.

Could we make sure that kong url and konnect URL are resilient to a trailing `/` provided by the user?

Reply:
Filed https://github.com/Kong/deck/issues/390